### PR TITLE
update notifications for one-click instances

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

<details>
<summary>Preview</summary>

## Before
image
image

## After

image
image

</details>

### CODE REVIEW

File: core/Notification/CoreNotifier.php [hunk 1]
```diff
@@ -78,10 +78,17 @@
...
-			$notification->setLink('https://nextcloud.com/enterprise/');
+			$notification->setParsedSubject($l->t('Nextcloud Enterprise subscription needed for more users'));
+			$notification->setParsedMessage($l->t('Enter your subscription key in the support app in order to increase the user limit. This will also grant you all additional benefits that Nextcloud Enterprise offers and is highly recommended for its operation in companies.'));
 			$notification->setIcon($this->url->getAbsoluteURL($this->url->imagePath('core', 'places/contacts.svg')));
+			$action = $notification->createAction();
+			$label = $l->t('Learn more ↗');
+			$link = 'https://nextcloud.com/enterprise/';
+			$action->setLabel($label)
+				->setParsedLabel($label)
+				->setLink($link, IAction::TYPE_WEB)
```
[the remark this answers is not in the document]
Comment (szaimen) [reply core/Notification/CoreNotifier.php:94]: fixed

**Review by come-nc — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
conflict though

**Pull request merged**